source/installation/index.rst

```diff
@@ -59,15 +59,28 @@ channel on Anaconda.org. The available packages are:
 - ``mantidworkbench``: General-purpose graphical-user-interface for plotting,
   writing scripts etc.
 
+It is worth noting that versions of Mantid after v6.7 (including the nightly) use
+Python 3.10. Mantid v6.7 and before use Python 3.8.
+
 Latest Release
 ##############
 
-To install any of the packages simply use the `-c` option in Conda to add the
-mantid channel and then the package name:
+To install Mantid into a new Conda environment use these options:
+- `-n` to specify the name of your new environment
```

Comment:
If you want this to be a bullet point list, I think this needs a whiteline above (at least it isn't showing on the preview!)